packages/compiler-dom/src/utils/element.ts

```diff
@@ -0,0 +1,33 @@
+const HTMLTag =
```

Comment:
Lets directly construct Sets with array literals here with `new Set(['html', ...])`

Reply:
Also, we will still need to perform a dev-only runtime check when users register components with reserved tag names (since those are compiled directly into native element calls, user components will be silently ignored if there is no such check). This check should be done in `setupStatefulComponent`.

Because of that, we probably also want to move these lists into `@vue/shared` so that both the runtime and the compiler can import them. (will be tree-shaken in the runtime because the check is dev only)

Reply:
> Lets directly construct Sets with array literals here with `new Set(['html', ...])`

Done.

Reply:
> Also, we will still need to perform a dev-only runtime check when users register components with reserved tag names (since those are compiled directly into native element calls, user components will be silently ignored if there is no such check). This check should be done in `setupStatefulComponent`.
> 
> Because of that, we probably also want to move these lists into `@vue/shared` so that both the runtime and the compiler can import them. (will be tree-shaken in the runtime because the check is dev only)

I got it.